src/main/java/util/Resources.java

```diff
@@ -0,0 +1,19 @@
+package util;
+
+import com.brunomnsilva.smartgraph.graphview.SmartGraphProperties;
+
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class Resources {
+
+    public static InputStream loadResourceFile(String relativePath) {
+        return Thread.currentThread().getContextClassLoader().getResourceAsStream(relativePath);
+    }
```

Comment:
Is there a benefit of getting the class loader this way instead of the more standard `Resources.class.getClassLoader()`? Does it prevent circumstances where `getClassLoader()` can return null in certain JVM implementations when the class loader is the system class loader?

Reply:
You are right, we'd better use the simpler approach Resources.class.getClassLoader() in this case, since we are developing a simple desktop application. Thread.currentThread().getContextClassLoader() is preffered in multi-module projects, libraries and in some cases of web applications.

I will test it and fix it on a next commit.